replica/client/client.go

```diff
@@ -392,6 +392,27 @@ func (c *ReplicaClient) LaunchReceiver(toFilePath string) (string, int32, error)
 	return c.host, reply.Port, nil
 }
 
+func (c *ReplicaClient) FileSync(fromServer *ReplicaClient, disks []string) error {
+	conn, err := grpc.Dial(c.syncAgentServiceURL, grpc.WithInsecure())
+	if err != nil {
+		return fmt.Errorf("cannot connect to SyncAgentService %v: %v", c.syncAgentServiceURL, err)
+	}
+	defer conn.Close()
+	syncAgentServiceClient := syncagentrpc.NewSyncAgentServiceClient(conn)
+
+	ctx, cancel := context.WithTimeout(context.Background(), GRPCServiceLongTimeout)
+	defer cancel()
+
+	if _, err := syncAgentServiceClient.FileSync(ctx, &syncagentrpc.FileSyncRequest{
```

Comment:
This FileSync RPC call should be async on the serverside and we can monitor the progress later before proceeding. And we don't know how long it will take to complete the sync so even `GRPCServiceLongTimeout` doesn't make sense.

Reply:
ok. Will use `context.Cancel(context.Background())` instead.

Reply:
I meant once the call client made to the server should just kick start the server work and return. I don't think `context.Background()` can do that. Can you check if this is indeed a blocking call?

Reply:
Context.background documentation says: `context.Background() returns a non-nil, empty Context. It is never canceled, has no values, and has no deadline.` 
I tested with 10G and 20G content. Both of them succeeded.

```
root@srao1:/mnt/shr# du -sh 
21G	.

curl http://172.104.18.252:31546/v1/volumes |  jq '.data[0].rebuildStatus'
[
  {
    "actions": null,
    "error": "",
    "isRebuilding": false,
    "links": null,
    "progress": 100,
    "replica": "t1-r-e14e83f3",
    "state": "complete"
  },
  {
    "actions": null,
    "error": "",
    "isRebuilding": false,
    "links": null,
    "progress": 0,
    "replica": "t1-r-d8b7cefc",
    "state": ""
  },
  {
    "actions": null,
    "error": "",
    "isRebuilding": false,
    "links": null,
    "progress": 0,
    "replica": "t1-r-f2ade456",
    "state": ""
  }
]
```